package/MDAnalysis/coordinates/chain.py

```diff
@@ -155,8 +242,84 @@ def __init__(self, filenames, **kwargs):
         #: source for trajectories frame (fakes trajectory)
         self.__chained_trajectories_iter = None
 
+        # calculate new start_frames to have a time continuous trajectory.
+        if continuous:
+            filetypes = np.unique([r.format for r in self.readers])
+            if not len(filetypes) == 1:
+                raise RuntimeError("Continuous only supported with all files "
+                                   "are from the same format. found {}".format(filetypes))
+            if np.any(np.array(n_frames) == 1):
+                raise RuntimeError("Need at least two frames in every trajectory")
+            if filetypes[0] not in ['XTC', 'TRR']:
```

Comment:
I think this should raise a `NotImplementedError` (or a `ValueError` as before, if there are technical limitations that prevent us from never being able to implement this for a format).